<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogt</class>
 <widget class="QDialog" name="Dialogt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tạo khóa</string>
  </property>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Tạo khóa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteUI">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chonSoNgTo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn số nguyên tố</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_P">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Q">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_P">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Q">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_chonSoNgTo">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn số ngTố</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_xoaAll">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa tất cả</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chonSoNgTo_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Tính các giá trị</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chonSoNgTo_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Khóa công khai</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chonSoNgTo_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Khóa bí mật</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_e">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_d">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_n">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bitKhoa">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>512</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2048</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4096</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_chonBitKhoa">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn số bit của khóa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_P_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_P_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_P_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
